crates/sui-framework/packages/bridge/sources/committee.move

```diff
@@ -164,9 +165,35 @@ module bridge::committee {
             registration
         };
 
+        // check uniqeuness of the bridge pubkey.
+        // `try_create_next_committee` will abort if bridge_pubkey_bytes are not unique and
+        // that will fail the end of epoch transaction (possibly "forever", well, we
+        // need to deploy proper validator changes to stop end of epoch from failing).
+        assert!(check_uniqueness_bridge_keys(self, bridge_pubkey_bytes), EDuplicatePubkey);
```

Comment:
I could just call the API here and have the `assert` inside `check_uniqueness_bridge_keys` 
if anybody feels strongly about it let me know

Reply:
assert inside `check_uniqueness_bridge_keys` is slightly preferential to me, but not a strong opinion

Reply:
changed